lib/MusicBrainz/Server/Report/ReleaseRGDifferentName.pm

```diff
@@ -0,0 +1,37 @@
+package MusicBrainz::Server::Report::ReleaseRGDifferentName;
+use Moose;
+
+with 'MusicBrainz::Server::Report::ReleaseReport',
+     'MusicBrainz::Server::Report::ReleaseGroupReport',
+     'MusicBrainz::Server::Report::FilterForEditor::ReleaseID';
+
+sub table { 'release_rg_different_name' }
+sub component_name { 'ReleaseRgDifferentName' }
+
+sub query {
+    "
+        SELECT
+            r.id AS release_id, rg.id AS release_group_id,
+            row_number() OVER (ORDER BY musicbrainz_collate(r.name), musicbrainz_collate(rg.name))
+        FROM
+            release r
+            JOIN release_group rg 
+            ON r.release_group = rg.id
+        WHERE (SELECT COUNT(*) FROM release WHERE release_group = rg.id) = 1
+          AND r.name != rg.name
+    "
+}
+
+
+__PACKAGE__->meta->make_immutable;
+no Moose;
+1;
+
+=head1 COPYRIGHT
+
+This file is part of MusicBrainz, the open internet music database.
```

Comment:
Should follow [coding style](https://musicbrainz.org/doc/Development/Server_Coding_Style#Perl_Module_Template_and_Documentation_Guidelines) instead.